doc/team_practices.md

```diff
@@ -0,0 +1,36 @@
+# C2 Team Practices
+
+This document outlines best practices for the C2 team, and anyone wanting to contribute to the project.
+
+## Workflow
+
+All code changes should originate in a story on the [story tracker](https://pivotaltracker.com/n/projects/1149728).
+
+Stories are clarified and prioritized in weekly iteration planning meetings (IPM). Team members self-select story ownership.
```

Comment:
Right now IPMs are every other week, hey? Weekly sounds good to me, just want to be sure everyone plays along